recipe/meta.yaml

```diff
@@ -1,5 +1,6 @@
 {% set name = "pynvml" %}
 {% set version = "11.5.3" %}
+{% set python_min = "3.6" %}
```

Comment:
As of PyNVML 11.5.3, [the Python minimum version was 3.6]( https://github.com/gpuopenanalytics/pynvml/blob/11.5.3/setup.py#L16 ). Hence using 3.6 above